text/0000-rename-distributed-ctx.md

```diff
@@ -0,0 +1,59 @@
+# Rename distributed context into tags and add support for Baggage.
+
+**Status:** `proposed`
+
+Rename Distributed-context to a less confusing term and restrict its use for observability.
+ 
+ 
+## Motivation
+
+Context Propagation is generalized to carry observability and non-observability contexts. This
+includes SpanContext, Distributed-context, Authentication-context, etc. Distributed-context
+carries key-value pairs for observability. The name Distributed-context is
+very confusing. Hence it should be changed to TagMap (or other better suggestion). 
+That is the first motivation.
+ 
+The second motivation is the restrict use of Distributed-context to observability. As it is
+written currently it does not specifically call out for using it only for non-observability purpose.
+However, it can be used to support 'Baggage' functionality for non-observability purpose as it is
+OpenTracing.
+There are few reasons to have different mechanism to address observability and
+non-observability use cases.
+1. For observability, key-value pairs SHOULD be write-only from application perspective. SDKs can read them
+   for example to apply them to measurements at the time of recording. For non-observability, 
+   key-value pairs SHOULD be writable/readable for application.
+2. For non-observability, it may require that the delivery is guaranteed and pairs are not dropped.
+3. Semantically two are different and mechanically it could be different as well.
+
+## Explanation
+
+We do following to
+- Rename distributed-context to TagMap.
```

Comment:
Please also rename `DistributedContext.Entry` to `Tag`, key and value to `TagKey`, `TagValue`.

Reply:
sure. My intention is to restore TagMap as defined in [OC](https://github.com/census-instrumentation/opencensus-specs/blob/master/tags/TagMap.md)